Libraries/Image/Image.android.js

```diff
@@ -188,6 +190,15 @@ const BaseImage = (props: ImagePropsType, forwardedRef) => {
     ref: forwardedRef,
   };
 
+  const objectFit = props.objectFit
```

Comment:
There should not be a prop for this style

Reply:
will update this thank you